src/firebase-rest.ts

```diff
@@ -36,9 +38,9 @@ type RequestOptions = {
 export class Client {
   private debug = false;
 
-  constructor(private appName: string,
-              private authToken?: string,
-              public uid?: string) {}
+  constructor(
+      private appName: string, private authToken?: string,
```

Comment:
strange choice of formatting but not a blocker

Reply:
I blame `clang-format`